File: mobile/lib/core/geofencing/models/geofence_zone.dart
/// Deterministic geofence zone classifications.
enum GeofenceZone {
  /// Outside both pre-warning (1,000 m) and core (500 m) envelopes.
  outside,

  /// Inside the informational pre-warning envelope.
  preWarning,

  /// Inside the acute core hazard envelope.
  core,
}

File: backend/src/main/java/com/geoshield/geofence/model/GeofenceZone.java
package com.geoshield.geofence.model;

/**
 * Deterministic geofence zone classifications.
 */
public enum GeofenceZone {
    /** Outside both the pre-warning and core hazard envelopes. */
    OUTSIDE,

    /** Inside the informational pre-warning envelope (1,000 m entry boundary). */
    PRE_WARNING,

    /** Inside the acute core hazard envelope (500 m entry boundary). */
    CORE
}
